tensorflow/contrib/tensorrt/resources/trt_allocator.h

```diff
@@ -59,6 +66,10 @@ class TRTDeviceAllocator : public nvinfer1::IGpuAllocator {
 
  private:
   tensorflow::Allocator* allocator_;
+
+  // supporting alignment from allocation request requires a map to free;
+  std::unordered_map<void*, void*> mem_map;
```

Comment:
Please add trailing underscore to the name. Same for `mem_pool`.

Reply:
Oops. :)